source/PlanetPanel.cpp

```diff
@@ -55,7 +56,7 @@ using namespace std;
 PlanetPanel::PlanetPanel(PlayerInfo &player, function<void()> callback)
 	: player(player), callback(callback),
 	planet(*player.GetPlanet()), system(*player.GetSystem()),
-	ui(*GameData::Interfaces().Get("planet"))
+	ui(*GameData::Interfaces().Get(Screen::Width() < 1280 ? "planet (small screen)" : "planet"))
```

Comment:
If the panel is initialized with a certain interface, it'll keep that interface even when the screen resolution changes. This impacts the PlanetPanel and SpaceportPanel, but not the other three panels, since they don't save an interface like this.